### DISCUSSION TITLE
The chat button is not appearing on the dashboard

### DISCUSSION THREAD

**Discussion by Azeem00786** [codemagic-ci-cd/codemagic-docs#2218] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

open code-magic dashboard 

### Expected results

chat button should be appear on dashboard same like shown in previous days

### Actual results

chat button is not appearing on the dashboard. please check the below image 
(https://github.com/codemagic-ci-cd/codemagic-docs/assets/91094418/499d4b33-c9ad-4abe-a8c5-73cd7913901d)

**Comment by szelemeh** [comment, marked as answer]
Hey @Azeem00786 ! I am not from Codemagic but this happened to me to, support chat button wasn't appearing even though I switched to paid account. Try turning off adblockers on Codemagic, it solved the issue for me.

**Reply by Azeem00786** [reply]
Thanks! @szelemeh
